dash-pipeline/bmv2/dash_routing_types.p4

```diff
@@ -193,4 +193,30 @@ action set_private_link_mapping(
     set_mapping_meter_attr(meta, meter_class, meter_class_override);
 }
 
-#endif /* _DASH_ROUTING_TYPES_P4_ */
\ No newline at end of file
+action set_private_link_nsg_mapping(
```

Comment:
the pl nsg mapping routing type is removed and the tunnel needs to be added in all routing types whenever the tunnel attribute is set.